source/apiVolontaria/location/models.py

```diff
@@ -0,0 +1,84 @@
+from django.db import models
+from django.core.exceptions import ValidationError
+
+
+class Country(models.Model):
+    """ Model for countries
+
+        iso_code corresponds to the ISO 3166 norm
+
+    """
+
+    iso_code = models.CharField(max_length=2, primary_key=True)
+    name = models.CharField(max_length=45, blank=False)
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name_plural = "Countries"
+        ordering = ["name", "iso_code"]
+        unique_together = (('iso_code', 'name'),)
+
+
+class StateProvince(models.Model):
+    """ Model for states and provinces
+
+        iso_code corresponds to the ISO 3166 norm
+
+    """
+
+    iso_code = models.CharField(max_length=3, primary_key=True)
```

Comment:
IDEM



Je pense que c'est plus durable dans le temps et plus lisible de specifier iso_code corresponds to the ISO 3166 norm comme un commentaire court directement au dessus du CharField().

Lorsqu'on viens chercher l'attribut, on ne va pas forcement penser a lire la documentation de la classe, on va parfois juste regarder l'attibut en lui meme.